_posts/2018-06-21-Rust-1.27.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: post
+title: "Announcing Rust 1.27"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.27.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.27.0 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.27.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1270-2018-06-21
+
+## What's in 1.27.0 stable
+
+This release has two big language features that people have been waiting for.
+But first, a small comment on documentation: All books in [the Rust
+Bookshelf] are [now searchable]! For example, here's [a search on TRPL for
```

Comment:
hm, should we make this "The Rust Programming Language"? Not all readers will be familiar with TRPL.